lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -48,7 +75,7 @@
     <button class="action action-cancel">
         <%=
             interpolate_text(
-                _.escape(gettext("Cancel {span_start}a new team{span_end}")),
+                _.escape(gettext("Cancel {span_start} team {span_end}")),
```

Comment:
@muzaffaryousaf @catong I'm not a huge fan of Cancel buttons to begin with. Is it absolutely necessary in this case?  Not hitting submit (create/update) is basically the same as Cancel.  I find that having cancel buttons only result in users mistakenly hitting the cancel button and swearing at their computers for having lost form data they worked hard to enter.  I'm in favor of just omitting it all together unless it is necessary for some reason.